Docs/user_guide/model_guidelines.rst

```diff
@@ -1,28 +1,30 @@
 :orphan:
 
-============================
+############################
 Model Guidelines for PyTorch
-============================
+############################
 
-To implement the Cross Layer Equalization API,  aimet_torch.cross_layer_equalization.equalize_model(),  AIMET creates a computing graph to analyze the sequence of Operations in the model.
-If your model is defined using certain constructs, it restricts AIMET from successfully creating and analyzing the computing graph. The following table lists the potential issues and workarounds.
+To implement the Cross Layer Equalization API,  `aimet_torch.cross_layer_equalization.equalize_model()`,  AIMET creates a computing graph to analyze the sequence of operations in the model.
 
-Note: These restrictions are not applicable, if you are using the **Primitive APIs**
+Certain model constructs prevent AIMET from creating and analyzing the computing graph. The following table lists these potential issues and workarounds.
 
+.. admonition NOTE::
+
+    These restrictions are not applicable if you are using the **Primitive APIs**.
 
 +------------------------+------------------------------+-----------------------------------+
-|     Potential Issue    | Description                  |     Work Around                   |
+|     Potential Issue    | Description                  |     Workaround                    |
 +========================+==============================+===================================+
 | ONNX Export            | Use torch.onnx.export()      | If ONNX export fails, rewrite the |
 |                        | to export your model.        | specific layer so that ONNX       |
 |                        | Make sure ONNX export passes | export passes                     |
 +------------------------+------------------------------+-----------------------------------+
 | Slicing Operation      |Some models use               | Rewrite the x.view() statement    |
 |                        |**torch.tensor.view()** in the| as follows:                       |
-|                        |forward function as follows:  | x = x.view(x.size(0), -1)         |
-|                        |x = x.view(-1, 1024)          |                                   |
-|                        |If view function is written   |                                   |
-|                        |as above, it causes an issue  |                                   |
+|                        |forward function as follows:  | `x = x.view(x.size(0), -1)`       |
```

Comment:
Can we reference python function using `:func:` to have better readability?
For example, :func:`x.view()`, :func:`x.view(x.size(0), -1)`

Reply:
Reformatted the page, including code samples.